contracts/SSVNetwork.sol

```diff
@@ -194,41 +180,35 @@ contract SSVNetwork is UUPSUpgradeable, Ownable2StepUpgradeable, ISSVNetwork {
             if (_validatorPKs[keccak256(publicKey)].owner != address(0)) {
                 revert ValidatorAlreadyExists();
             }
-            _validatorPKs[keccak256(publicKey)] = Validator({
-                owner: msg.sender,
-                active: true
-            });
+            _validatorPKs[keccak256(publicKey)] = Validator({owner: msg.sender, active: true});
         }
 
-        bytes32 hashedCluster = keccak256(
-            abi.encodePacked(msg.sender, operatorIds)
-        );
-        {
-            bytes32 hashedClusterData = keccak256(
+        bytes32 hashedCluster = keccak256(abi.encodePacked(msg.sender, operatorIds));
+
+        if (clusters[hashedCluster] == bytes32(0)) {
+            cluster = Cluster({validatorCount: 0, networkFeeIndex: 0, index: 0, balance: 0, active: true});
```

Comment:
@lior-blox Do we want to force active: true here?  If the user registers first cluster with false we will revert it to hardcoded true

Reply:
Actually maybe this is better solution

if (clusters[hashedCluster] == bytes32(0)) {
            if (!cluster.active) {
                revert ActiveMustBeTrue
            }
            cluster = Cluster({validatorCount: 0, networkFeeIndex: 0, index: 0, balance: 0, active: true});
        }